tests/Mocks/Servers.h

```diff
@@ -58,7 +58,13 @@ namespace mocks {
 /// @brief mock application server with no features added
 class MockServer {
  public:
-  MockServer();
+  // Note, setting injectClusterIndexes causes the "create" methods to fail.
+  // this is all hardly worked around the Cluster engine and needs a proper
+  // clean up. It is highly recommended to not set injectClusterIndexes unless
+  // you want to specificly test something that selects an index, but cannot use
+  // it. Use with care for now.
```

Comment:
IDK if possible, but could we somehow print a warning for developers, which might not read that HINT? 
If not (e.g. jenkins is not happy), ignore for now. But could help in case any developer might skip that section.

Reply:
Ah, I think it can be ignored as it is false by default.